config/config.sample.php

```diff
@@ -2400,9 +2400,9 @@
 'reference_opengraph' => true,
 
 /**
- * Enable use of old unified search
+ * Disable features that are do respect accessibility standards yet.
```

Comment:
To be aligned with the property name (not `enable_*` for "disabling").

```suggestion
 * Enable features that don't respect accessibility standards yet.
```

Reply:
However, IMO, `'disable_non-accessible_features' => false"` config by default would fit better, so default while would be the same as false and the focus is on disabling.

Reply:
Well I would say from a language perspective:

* `true` should mean `ON`
* `false` should mean `OFF`

So a description of disable+false translates to "enable" and that is hard to process for humans.

Reply:
Yes, thanks for the correction, I started with `disable...` and switched to `enable..` because of @AndyScherzinger explication, but I forgot to adjust the comment.